docs/clients/promtail/configuration.md

```diff
@@ -1159,3 +1188,34 @@ scrape_configs:
       - source_labels: ['__syslog_message_hostname']
         target_label: 'host'
 ```
+
+## Example Push Config
+
+The example starts Promtail as a Push receiver and will accept logs from other Promtail instances or the Docker Logging Dirver:
+
+```yaml
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: http://ip_or_hostname_where_Loki_run:3100/loki/api/v1/push
+
+scrape_configs:
+- job_name: push1
+  push:
+    server:
+      http_listen_port: 3500
+      grpc_listen_port: 3600
+    labels:
+      pushserver: push1
```

Comment:
the already existing `external_labels` are applied in the promtail client which is downstream from the `Handle` function all our targets call, so those labels will be applied to any log line promtail sends and can be sent via command line.

Not sure what you mean about the `filename` label though? In this case we won't have a `filename` label because we aren't reading from a file?